Sasquatch/Sasquatch/Auth/Auth0Provider.swift

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Foundation
+import AppCenter
+import Auth0
+
+class Auth0Provider : NSObject, AuthProviderDelegate {
+
+  private let audienceDomain = "https://appcentersdk.auth0.com/userinfo"
+  private var credentialsManager = CredentialsManager(authentication: Auth0.authentication())
+
+  func signIn(_ completionHandler: @escaping (MSUserInformation?, Error?) -> Void) {
+    Auth0.webAuth().scope("openid offline_access").audience(audienceDomain).start {
+      switch $0 {
+      case .failure(let error):
+        completionHandler(nil, error)
+      case .success(let credentials):
+        let userInformation = MSUserInformation()
+        userInformation.accessToken = credentials.accessToken
```

Comment:
this is not JWT, what are we doing with it?

Reply:
Nothing.

Reply:
It will be removed in another PR.